src/app/clients/add/formSections/NumberAdultsCard.tsx

```diff
@@ -64,24 +71,27 @@ const NumberAdultsCard: React.FC<CardProps> = ({
             required={true}
             text="Please enter the number of adults (aged 16 or above) in the appropriate category."
         >
-            <GappedDiv>
+            <>
                 <FreeFormTextInput
                     error={errorExists(formErrors.adults)}
                     label="Female"
+                    defaultValue={getNumberAdultsDefault(fields.adults, "female")}
                     onChange={getNumberAdults(fieldSetter, errorSetter, fields.adults, "female")}
                 />
                 <FreeFormTextInput
                     error={errorExists(formErrors.adults)}
                     label="Male"
+                    defaultValue={getNumberAdultsDefault(fields.adults, "male")}
                     onChange={getNumberAdults(fieldSetter, errorSetter, fields.adults, "male")}
                 />
                 <FreeFormTextInput
                     error={errorExists(formErrors.adults)}
                     helperText={errorText(formErrors.adults)}
                     label="Prefer Not To Say"
+                    defaultValue={getNumberAdultsDefault(fields.adults, "other")}
                     onChange={getNumberAdults(fieldSetter, errorSetter, fields.adults, "other")}
                 />
-            </GappedDiv>
+            </>
```

Comment:
How come we don't need the gapped div anymore out of interest?

Reply:
deleted by accident :)